include/seqan3/alignment/pairwise/detail/pairwise_alignment_algorithm_banded.hpp

```diff
@@ -71,22 +66,10 @@ class pairwise_alignment_algorithm_banded :
      *
      * Initialises the algorithm given the user settings from the alignment configuration object.
      *
-     * \throws seqan3::invalid_alignment_configuration if the given band settings are invalid.
+     * \throws seqan3::invalid_alignment_configuration.
```

Comment:
```suggestion
```
?

Reply:
Well, it still throws, but it happens in the base policy constructor now.